doc/reference/peripherals/peci.rst

```diff
@@ -0,0 +1,28 @@
+.. _peci:
+
+PECI
+####
+
+Overview
+********
+The Platform Environment Control Interface, abbreviated as PECI,
+is a thermal management standard introduced in 2006
+with the Intel Core 2 Duo microprocessors.
+The PECI interface for external devices to read processor temperature,
```

Comment:
The PECI interface allows external devices to read processor temperature, bla bla